aporeto-inc/trireme-lib
# Implements UIDPAM feature.

#### Description
* This PR  implements UIDPAM feature that allows one to launch protected processes based on the userid. 

#### Test plan

1. Launch 2 UID servers and try connecting to them via curl launched from different UIDS. The access should be granted as per the network policy.
2. Use apoctl to connect to UID user.
3. Run a server in docker container and try to connect to it via curl/apoctl .
4. Launch an UID server process and try to connect from the docker with appropriate policies configured.
> Fixes #.
